mmv1/third_party/terraform/services/securitycenter/resource_scc_folder_big_query_export_config_test.go

```diff
@@ -0,0 +1,124 @@
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {},
+			"time":   {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterFolderBigQueryExportConfig_basic(context),
+			},
+			{
+				ResourceName:            "google_scc_folder_scc_big_query_exports.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"update_time"},
+			},
+			{
+				Config: testAccSecurityCenterFolderBigQueryExportConfig_update(context),
+			},
+			{
+				ResourceName:            "google_scc_folder_scc_big_query_exports.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"update_time"},
+			},
+		},
+	})
+}
+
+func testAccSecurityCenterFolderBigQueryExportConfig_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_folder" "folder" {
+  parent       = "organizations/%{org_id}"
+  display_name = "%{folder_name}"
+  deletion_protection = false
+}
+resource "google_bigquery_dataset" "default" {
+  dataset_id                  = "%{dataset_id}"
+  friendly_name               = "test"
+  description                 = "This is a test description"
+  location                    = "US"
+  default_table_expiration_ms = 3600000
+  default_partition_expiration_ms = null
+  labels = {
+    env = "default"
+  }
+  lifecycle {
+	ignore_changes = [default_partition_expiration_ms]
+  }
+}
+resource "time_sleep" "wait_1_minute" {
+	depends_on = [google_bigquery_dataset.default]
+	create_duration = "3m"
+}
+resource "google_scc_folder_scc_big_query_exports" "default" {
+  big_query_export_id    = "%{big_query_export_id}"
+  folder 	   = google_folder.folder.folder_id
+  dataset      = google_bigquery_dataset.default.id
+  description  = "Cloud Security Command Center Findings Big Query Export Config"
+  filter       = "state=\"ACTIVE\" AND NOT mute=\"MUTED\""
+
+  depends_on = [time_sleep.wait_1_minute]
+}
+
+`, context)
+}
+
+func testAccSecurityCenterFolderBigQueryExportConfig_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_folder" "folder" {
+  parent       = "organizations/%{org_id}"
+  display_name = "%{folder_name}"
+  deletion_protection = false
+}
+resource "google_bigquery_dataset" "default" {
+  dataset_id                  = "%{dataset_id}"
+  friendly_name               = "test"
+  description                 = "This is a test description"
+  location                    = "US"
+  default_table_expiration_ms = 3600000
+  default_partition_expiration_ms = null
+  labels = {
+    env = "default"
+  }
+  lifecycle {
+	ignore_changes = [default_partition_expiration_ms]
+  }
+}
+resource "google_scc_folder_scc_big_query_exports" "default" {
+  big_query_export_id    = "%{big_query_export_id}"
+  folder 	   = google_folder.folder.folder_id
+  dataset      = google_bigquery_dataset.default.id
```

Comment:
note: if this _is_ required, make sure it gets updated here as well.

Reply:
Got to know that the fields are mandatory, so addressed them in the yaml file as required.